packages/@react-spectrum/actionbar/src/ActionBarContainer.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {DOMRef} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+import React from 'react';
+import {SpectrumActionBarContainerProps} from '@react-types/actionbar';
+import {useProviderProps} from '@react-spectrum/provider';
+
+
+function ActionBarContainer(props: SpectrumActionBarContainerProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
So my understanding of this component based on the issue is that even if the Table is given a height, the ActionBarContainer height should not change.
As such, we may need for the final dom to look something like this
```
<ActionBarContainer> -> display: flex, flexDirection: column
  <Table/> -> flex: 1 1 auto;
  <ActionBar/> -> flex: 0 0 auto;
</ActionBarContainer>
```

To automatically get those styles set on the components inside the ActionBarContainer, we may need to use Slots. There are examples of Slots usage in Dialog https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/dialog/src/Dialog.tsx#L97 as well as docs discussing usage of already implemented ones https://react-spectrum.adobe.com/react-spectrum/layout.html#slots